resources/js/components/catalog-new/NewYearSlider.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div class="range-slider py-3">
+        <div class="col-xs-6 col-sm-1 text-left text-sm-right">
+            <input
+                class="sans range-slider-from"
+                maxlength="5"
+                pattern="[-]?[0-9]{1,4}"
+                step="5"
+                @change="touched = true"
+                v-model.lazy="tempYearRange[0]"
+            />
+        </div>
+        <div class="col-xs-6 col-sm-1 col-sm-push-10 text-right text-sm-left">
+            <input
+                class="sans range-slider-to"
+                maxlength="5"
+                pattern="[-]?[0-9]{1,4}"
+                step="5"
+                @change="touched = true"
+                v-model.lazy="tempYearRange[1]"
+            />
+        </div>
+        <div class="col-xs-12 col-sm-10 col-sm-pull-1">
+            <div class="pt-1" :id="filterName + '-slider'">
+                <slider
+                    v-model="tempYearRange"
+                    :min="getMinMax().min || 0"
+                    :max="getMinMax().max || 30"
+                    tooltip="none"
+                    :duration="0"
+                    :dotSize="16"
+                    lazy
+                    @change="submitForm"
+                    @drag-end="submitForm"
+                    @dragging="updateYearRange"
+                >
+                </slider>
+                <fieldset :disabled="!touched">
+                    <input
+                        :id="filterName"
+                        :name="filterName"
+                        type="hidden"
+                        v-bind:value="tempYearRange.join(',')"
+                    />
+                </fieldset>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import VueSlider from 'vue-slider-component'
+
+export default {
+    props: {
+        yearRange: Array,
+        filter: Object,
+        filterName: String,
+    },
```

Comment:
These prop names are not very clear to me:

* is `yearRange` the boundary interval, or is it current value?
* Why does a slider need a `filter` or even `filterName`?

I would probably expect something like this
```suggestion
    props: {
        min: Number,
        max: Number,
        defaultValue: Object,
    },
```
```suggestion
    props: {
        min: Number,
        max: Number,
        defaultFrom: Number,
        defaultTo: Number,
    },
```